docs/developing/monitoring.rst

```diff
@@ -0,0 +1,91 @@
+.. _Monitoring:
+
+Monitoring
+----------
+
+Steamship enables easy monitoring of the health and behavior of your running
+:ref:`Packages` and :ref:`Plugins`.
+
+
+Monitoring via Web
+~~~~~~~~~~~~~~~~~~
+
+Steamship's website provides a hub for monitoring your packages and plugins. The monitoring page for
+an instance provides application logs, a detailed breakdown of usage, and a view of related
+:ref:`Tasks`.
+
+To access the monitoring page for an instance, visit the instance page. For example, for a package instance,
```

Comment:
Longer term I wonder if we'd be benefited by having a page called `Instance Page` that explains what it is and how to get there, and then we can just use the term Instance page as a link to that page